Wikipedia.js

```diff
@@ -120,7 +120,14 @@ function doWeb(doc, url) {
 					item.date = pages[i].touched;
 				}
 				
-				item.title = pages[i].displaytitle;
+				// https://github.com/zotero/zotero/blob/master/chrome/content/zotero/bindings/styled-textbox.xml
```

Comment:
styled-textbox.xml isn't relevant here — these are getting saved to a plain text field. All that matters is the [subset of HTML](https://www.zotero.org/support/kb/rich_text_bibliography) allowed in citeproc-js. We could allow other things if we convert them to those (e.g., `<strong>` → `<b>`, `<em>` → `<i>`), but otherwise we don't want to save them.

Reply:
Thanks, fixed this + the test regressions. I was confused between the two lists of allowed tags and figured the superset was correct.